cpp/autosar/test/rules/A15-4-2/test.cpp

```diff
@@ -21,4 +23,13 @@ void test_indirect_throw_3() noexcept(false) {
 
 void test_indirect_throw_4() {
   throwA(); // COMPLIANT - function marked as noexcept(false)
+}
+
+void test_indirect_throw_5() noexcept {
+  noexceptIndirectThrowA(); // COMPLIANT - noexceptIndirectThrowA would call
```

Comment:
A similar FN exists as for EXP55-CPP when a `std::bad_exception` can still be thrown.

Reply:
See below for rationale on not adding this case.